streaming/java/streaming-runtime/src/main/java/org/ray/streaming/runtime/worker/context/RayRuntimeContext.java

```diff
@@ -2,9 +2,20 @@
 
 import static org.ray.streaming.util.Config.STREAMING_BATCH_MAX_COUNT;
 
+import com.google.common.base.Preconditions;
 import java.util.Map;
 import org.ray.streaming.api.context.RuntimeContext;
 import org.ray.streaming.runtime.core.graph.ExecutionTask;
+import org.ray.streaming.state.backend.KeyStateBackend;
+import org.ray.streaming.state.backend.OperatorStateBackend;
+import org.ray.streaming.state.backend.TransactionKeyStateBackend;
+import org.ray.streaming.state.keystate.desc.AbstractStateDescriptor;
+import org.ray.streaming.state.keystate.desc.ListStateDescriptor;
+import org.ray.streaming.state.keystate.desc.MapStateDescriptor;
+import org.ray.streaming.state.keystate.desc.ValueStateDescriptor;
+import org.ray.streaming.state.keystate.state.ListState;
+import org.ray.streaming.state.keystate.state.MapState;
+import org.ray.streaming.state.keystate.state.ValueState;
 
 /**
  * Use Ray to implement RuntimeContext.
```

Comment:
RayRuntimeContext sounds like the runtimeContext of Ray Core itself. RayStreamRuntimeContext might be a better choice that developer could understand instantly while reading the code.